components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -2063,12 +2063,12 @@ spec:
                       cpu: 100m
                       memory: 1Gi
                 topologySpreadConstraints:
-                  - maxSkew: 1
-                    topologyKey: topology.kubernetes.io/zone
-                    whenUnsatisfiable: DoNotSchedule
-                    labelSelector:
-                      matchLabels:
-                        app: tekton-pipelines-controller
+                - labelSelector:
+                    matchLabels:
+                      app: tekton-pipelines-controller
+                  maxSkew: 1
+                  topologyKey: topology.kubernetes.io/zone
+                  whenUnsatisfiable: DoNotSchedule
```

Comment:
Regenerating the files using script change the order as when this was added, it was not done with the script but this is a no-op change, only order is changing